### PR TITLE
Feat/astro v2

### PR DESCRIPTION
**Pull request by DBozhinovski**
## Summary of change

Updates the Astro integration. Updates its readme to clarify why this particular integrations works the way it does. Adds alternatives to README.

## Checklist for important updates

-   [x] Changelog has been updated
-   [x] Changes to the version if needed
    -   In `package.json`
    -   In `package-lock.json`
    -   In `lib/ts/version.ts`
-   [x] Had run `npm run build-pretty`
-   [x] Had installed and ran the pre-commit hook

### CODE REVIEW

File: lib/ts/version.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const package_version = "0.0.49";
+export const package_version = "0.0.50";
```
Comment (rishabhpoddar) [inline lib/ts/version.ts:1]: no need to change versions since only boilerplate code has changed

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,15 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
-## [0.0.49] - 2024-09-10
+## [0.0.50] - 2024-08-22
+
+-   Updates Astro integration
```
Comment (rishabhpoddar) [inline CHANGELOG.md:12]: can remove this since no version change is needed

File: boilerplate/fullstack/astro/src/config/appInfo.ts [hunk 3]
```diff
@@ -3,6 +3,6 @@ export const appInfo = {
     appName: "SuperTokens Astro demo app",
     apiDomain: "http://localhost:4321",
     websiteDomain: "http://localhost:4321",
```
Comment (rishabhpoddar) [inline boilerplate/fullstack/astro/src/config/appInfo.ts:5]: what about the issue one person was saying that choosing emailpassword + social login only makes an app with social login?

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: boilerplate/fullstack/astro/src/config/appInfo.ts (hunk 3 above)
Comment (DBozhinovski) [reply boilerplate/fullstack/astro/src/config/appInfo.ts:5]: image

Seems to work fine.

Comment (DBozhinovski) [reply boilerplate/fullstack/astro/src/config/appInfo.ts:5]: Tried it with multiple combinations (just social, just email/pass, social + paswordless, social + email/pass, all three at once) too.

**Pull request merged**